ov/storage_volume_template.go

```diff
@@ -232,6 +232,7 @@ func (c *OVClient) UpdateStorageVolumeTemplate(sVolTemplate StorageVolumeTemplat
 
 	log.Debugf("REST : %s \n %+v\n", uri, sVolTemplate)
 	_, err := c.RestAPICall(rest.PUT, uri, sVolTemplate)
+
```

Comment:
Unnecessary changes. Why empty line is added?

Reply:
removed.